app/helpers/application_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module ApplicationHelper
+  def service_name_for(current_user = nil)
+    return "Mentoring for early career teachers" if current_user&.mentor?
```

Comment:
Let's name the variable `user` or something like that - `current_user` is also a helper method from devise, available in a lot of places, and I don't want to confuse the two.